src/draw.m

```diff
@@ -153,5 +162,28 @@ function draw_state_density(t_now)
 
 end
 
+function draw_vaccination_density(t_now)
+    if isempty(findobj('Type', 'Figure', 'Name', 'Vaccination'))

```

Comment:
Can you just change to this :+1: 
` if isempty(findobj('Type', 'Figure', 'Name', 'vaccination_density'))`
As it is written in the case above. It's just because figure with name `vaccination` already exists and would produce some errors if plotted together with this one. And because for saving, i will save the figures according to their name (see my pull request)

Reply:
done